experimental/client-app/README.md

```diff
@@ -13,17 +13,81 @@ Please have [node](https://nodejs.org/en/ ) 14 & npm 8 or higher installed.
 
 ## Get started
 
-Within your terminal, open one tab to run the proxy and another to run the app.
+### Run web app locally
 
 ```bash
 npm install
 npm build 
-npm run dev
+npm run start
+```
+
+Open your browser to `localhost:8000`.
+
+### Run auth api locally
+
+```bash
+npm install
+npm run start
+```
+
+Server will run at `localhost:4000`.
+
+
+## Deploying containers
+
+```bash
+export PROJECT_ID=<YOUR_PROJECT_ID>
+```
+
+Requires the following environment variables to be set within each Cloud Run container.
+
+| Env Vars       | Description                                 | Example                                    |
+| -------------- | ------------------------------------------- | ------------------------------------------ |
+| API_URL        | path that points to server api              | `https://content-api-abc123-uc.a.run.app`  |
+| SITE_URL       | path that points to current client web app  | `https://lit-website-abc123-uc.a.run.app`  |
+| AUTH_API_URL   | path that points to nodejs auth api url     | `https://lit-auth-api-abc123-uc.a.run.app` |
+| REDIRECT_URI   | path that points to AUTH_API_URL callback   | `${AUTH_API_URL}/auth/google`              |
+
+| Secrets        | Description                                                       |                                                                                  |
+| -------------- | ----------------------------------------------------------------- | -------------------------------------------------------------------------------- |
+| JWT_SECRET     | String to encode jwt token (only required for auth api container) | `random-value-emblem-string`                                                     |
```

Comment:
Is `random-value-emblem-string` a realistic value? (`some-arbitrary-string` might be a tad clearer)